lib/arc.d.ts

```diff
@@ -509,14 +651,15 @@ declare module "daostack-arc-js" {
      */
     proposeContributionReward(
       opts: ProposeContributionParams
-    ): Promise<TransactionReceiptTruffle>;
-    setParams(params: ContributionRewardParams): Promise<string>;
+    ): Promise<ArcTransactionProposalReturnType>;
+
+    setParams(params: ContributionRewardParams): Promise<ArcTransactionResultReturnType>;
 
     /**
      * Event functions as defined by the parent Truffle contract
      */
-    LogNewContributionProposal(filters : any, options : any) : any;
-    LogProposalExecuted(filters : any, options : any) : any;
-    LogProposalDeleted(filters : any, options : any): any;
+    LogNewContributionProposal(filters: any, options: any): any;
```

Comment:
I believe the names of these have changed in the latest Arc

Reply:
We can change them when we pull in a new version of arc.  I don't want to play another game of moving target with arc like I had to do with the last mammoth PR.